core/kazoo_endpoint/src/kz_endpoint.erl

```diff
@@ -1552,6 +1554,11 @@ maybe_set_call_forward({Endpoint, Call, CallFwd, CCVs}) ->
                        )
     }.
 
+-spec maybe_set_rtcp_mux(ccv_acc()) -> ccv_acc().
+maybe_set_rtcp_mux({Endpoint, Call, CallFwd, CCVs}) ->
+    RTCP_MUX = kz_json:is_true([<<"media">>, <<"rtcp_mux">>], Endpoint, ?DEFAULT_RTCP_MUX),
+    {Endpoint, Call, CallFwd, kz_json:set_value(<<"RTCP-MUX">>, RTCP_MUX, CCVs)}.
```

Comment:
@lazedo i reviewed PR according last recommendation in https://github.com/2600hz/kazoo/pull/3872
But leaved default value because case got 'undefined' if default value is not present in system_config